Documentation/specs/runtime-configuration-file.md

```diff
@@ -287,52 +304,3 @@ When setting up the TPA and native library lists, it will do the following for t
 Note one important aspect about asset resolution: The resolution scope is **per-package**, **not per-application**, **nor per-asset**. For each individual package, the most appropriate RID is selected, and **all** assets taken from that package must match the selected RID exactly. For example, if a package provides both a `linux-x64` and a `unix` RID (in the `ubuntu.14.04-x64` example above), **only** the `linux-x64` asset would be selected for that package. However, if a different package provides only a `unix` RID, then the asset from the `unix` RID would be selected.
 
 The path to a runtime-specific asset is resolved in the same way as a normal asset (first check Servicing, then Package Cache, App-Local, Global Packages Location, etc.) with **one exception**. When searching app-local, rather than just looking for the simple file name in the app-local directory, a runtime-specific asset is expected to be located in a subdirectory matching the relative path information for that asset in the lock file. So the `native` `sni.dll` asset for `win7-x64` in the `System.Data.SqlClient` example above would be located at `APPROOT/runtimes/win7-x64/native/sni.dll`, rather than the normal app-local path of `APPROOT/sni.dll`.
-
```

Comment:
I thought that this section still basically applies, it's just that now it also applies to the frameworks …
In which case we should keep this, just tweak it to make it clear it works in frameworks as well.

Reply:
The difference are that the values are no longer "absolute" meaning completely override whatever the framework had. Instead, they use the "most restrictive"

Reply:
We should add description of this behavior then. Either replace the section you removed here, or extend the description of the `framework` property [here](https://github.com/dotnet/cli/pull/9995/files#diff-b65c14694db4c6fab803eacceb1b5bedL126).

Reply:
I am not sure if this document is the best location for this information, or whether it should be placed in the core-setup repo like the existing design for this feature is. In either case, I did re-add that section and tweak it for current semantics.

Reply:
I agree from CoreCLR developer point of view. Unfortunately this doc is the only real documentation we have about this. We have an [issue](https://github.com/dotnet/docs/issues/2645) about the lack of docs... but that will probably take a while to fix. So for now I prefer to keep everything in one place so that we can point users to at least doc.